mineral-ui/mineral-ui
# [Card] Add CardActions, CardDivider, CardFooter; update styles

### Description


- Add CardActions, CardDivider, CardFooter
- Add actions, avatar, and variant to CardTitle
- Add icons used within Card components
- Various style updates to all Card components
- Remove/update component theme variables
- Add/update global theme variables
- Update website demos and content
- Update snapshots
- Remove no-longer-needed customization in Popover
- Fix small layout bug in website's Page (necessary to allow clearing of floats inside examples)

### Motivation and context

Closes #470 

### Screenshots, videos, or demo, if appropriate


https://470-card-updates--mineral-ui.netlify.com/components/card

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to Card (and CardActions, CardBlock, CardFooter, etc...)
3. Confirm all necessary functionality
4. Check all demo content for clarity, consistency, and errors

### Types of changes

- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

Breaking:

- **Card**: `CardRow_margin` & `CardRow_padding` theme variables changed to `CardRow_marginVertical` & `CardRow_paddingHorizontal`, respectively
- **CardTitle**: Removed `minor` prop and all associated theme variables; significant change to usage of `meta` prop; removed `CardTitle_marginTop` theme variable

### Checklist




### How does this PR make you feel?

![Card haircut](https://media.giphy.com/media/f45WwsBiZ0oIE/giphy.gif)
